### DISCUSSION TITLE
ST7735 for 2.4” displays?

### DISCUSSION THREAD

**Discussion by StuMunsonMcGee** [peterhinch/micropython-nano-gui#75] [Q&A]
Does the ST7735R driver work for a 2.4” 240x330 TFT display?  I’ve gotten it to work for 1.8” 128x160 and 1.44” 128x128 displays. But the larger display lights up and tries to display the image (just text from CWriter) but fails. Any help would be greatly appreciated

**Comment by peterhinch** [comment]
The ST7735r chip is only capable of driving displays up to 160x130 pixels. Your 240x**320** display (surely 320, not 330) must use a different chip. Possibly ILI9341?
